crates/typst-syntax/src/file.rs

```diff
@@ -57,6 +57,23 @@ impl FileId {
         id
     }
 
+    /// Create a new "fake" file specification which is *not* interned.
```

Comment:
Not interned seems inaccurate when we have `interner.from_id.push(..)`. Maybe something like:

Create a new unique file specification, which is not otherwise accessible.